===================================================================
RCS file: /cvs/xorg/xc/lib/Xft/xftint.h,v
retrieving revision 1.1
retrieving revision 1.4
diff -u -r1.1 -r1.4
--- xc/lib/Xft/xftint.h	2003/11/14 16:48:48	1.1
+++ xc/lib/Xft/xftint.h	2004/08/13 19:03:15	1.4
@@ -40,6 +43,16 @@
 #include <fontconfig/fcprivate.h>
 #include <fontconfig/fcfreetype.h>
 
+#ifndef HAVE_CONFIG_H
+# if (FREETYPE_MAJOR > 2 ||						    \
+      (FREETYPE_MAJOR == 2 && (FREETYPE_MINOR > 1 ||			    \
+			       (FREETYPE_MINOR == 1 && FREETYPE_PATCH >= 5))))
+#  define HAVE_FT_BITMAP_SIZE_Y_PPEM 1
+# else
+#  define HAVE_FT_BITMAP_SIZE_Y_PPEM 0
+# endif
+#endif
+
 typedef struct _XftMatcher {
     char    *object;
     double  (*compare) (char *object, FcValue value1, FcValue value2);
